### PR TITLE
Images/util

### PR DESCRIPTION
**Pull request by festinalente91**
- add image util
- you can get color hex data from image url.
- you can get text from image

### CODE REVIEW

File: src/server/util/images.util.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
+import fetch from 'node-fetch';
+
+const CONFIG = {
+  GOOGLE: {
+    API_KEY: process.env.GOOGLE_API_KEY
```
Comment (ochanje210) [inline src/server/util/images.util.js:5]: where is this process coming from?

Comment (lyw0149) [reply src/server/util/images.util.js:5]: from external env file.

Comment (ochanje210) [reply src/server/util/images.util.js:5]: ok so it is global?

Comment (lyw0149) [reply src/server/util/images.util.js:5]: yup

File: src/server/util/images.util.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        return fetch(address, {
+          method: 'POST',
+          headers: {
+            'content-type': 'application/json'
+          },
+          body: JSON.stringify(body)
+        })
+        .then(res => {
+          if (!res.ok) {
+            res.json().then(console.log);
```
Comment (lyw0149) [inline src/server/util/images.util.js]: console.log

File: src/server/util/images.util.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+            throw Error('No result');
+          }
+          return res.json();
+        })
+        .then(obj => obj.responses[0].imagePropertiesAnnotation.dominantColors.colors[0].color)
+        .catch((err) => {
+          console.log(err);
```
Comment (lyw0149) [inline src/server/util/images.util.js]: console.log

**Review by lyw0149 — APPROVED** [review]
LGTM

**Pull request merged**